dvc/command/config.py

```diff
@@ -114,8 +114,8 @@ def add_parser(subparsers, parent_parser):
         action="store_true",
         help="Unset option.",
     )
-    config_parser.add_argument("name", help="Option name.")
+    config_parser.add_argument("name", help="Setting name.")
```

Comment:
Same here, `config` is about `config options`, not `settings`. For example, take a look at `git config --help`.

Reply:
Yes I see that outputs ```usage: git config [<options>]
``` so would need to also change that to `<setting>`... OK well, this PR was just a suggestion, we can close it and I'll go back to figure out another way to clarify the double use of the term "options" in https://dvc.org/doc/commands-reference/config

Reply:
ah, k. I see it now. How is Git doing it their docs - they should have the same problem - command line options + config options

Reply:
Git also "overloads" the term "options" all over the [git-config](https://git-scm.com/docs/git-config) command description, but calls Git options `<file-option>` in the ref. **SYNOPSYS**.

Reply:
I'm going to close this PR but  please lmk if we want to call them "file-options" too. I'm probably going to fo with "DVC config options" or just "config options" for now.